tests/src/lib.rs

```diff
@@ -1,219 +0,0 @@
-extern crate bridge;
```

Comment:
why is it removed?

Reply:
those should indeed be commented in again and adapted to the changes. will do